admin/settings/class-admin-apple-settings-section.php

```diff
@@ -10,302 +10,6 @@
 class Admin_Apple_Settings_Section extends Apple_News {
 
 	/**
-	 * All available iOS fonts.
-	 *
-	 * @since 0.4.0
-	 * @var array
-	 * @access protected
-	 */
-	protected static $fonts = array(
```

Comment:
The font list was moved to the `Theme` object to keep it with the formatting settings migrated there.